netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
 publish = "docs/build"
-command = "make netlify"
+command = "LATEST_TAG=$(git describe --tags --abbrev=0) && git checkout $LATEST_TAG && BUILD_VERSION=$LATEST_TAG make netlify"
```

Comment:
Hmm perhaps we could put that into the Makefile? We could have two different targets, like `make netlify` vs `make netlify-preview` 🤔

Reply:
Yeah I am just having a fiddle with some different options at the moment. Not set on doing it this way, just trying to see if it works.

Reply:
This has been addressed now 🙂